tests/numpy/test_arithmetic.py

```diff
@@ -126,9 +144,48 @@ def test_inner_outer(nps_app_inst):
     assert np.allclose(np.inner(A, B), nps.inner(nps_A, nps_B).get())
     assert np.allclose(np.outer(A, B), nps.outer(nps_A, nps_B).get())
 
+def test_broadcast(nps_app_inst):
```

Comment:
This is good, but I think this is missing coverage for all variations of the parameter `axes`. Can you add tests that cover the different types of `axes` we don't support?

Since this is an error test, can you rename the test to `test_broadcast_error`?